docs/tailwind.config.ts

```diff
@@ -0,0 +1,11 @@
+import { orbitComponentsPreset } from "@kiwicom/orbit-tailwind-preset";
+import type { Config } from "tailwindcss";
+
+export default {
+  content: ["./src/**/*.{js,jsx,ts,tsx}"],
```

Comment:
This is actually failing to inspect Orbit components using on those files. It is only checking for tailwind classes on the src documentation files. But it needs to go deeper and check the Orbit components.
So this should be
```suggestion
  content: [
    "./src/**/*.{js,jsx,ts,tsx}",
    path.join(path.dirname(require.resolve("@kiwicom/orbit-components")), "**/*.js"),
  ],
```